tools/test_utils/utilities.py

```diff
@@ -1,30 +1,44 @@
 import argparse
+import asyncio 
+import os
+import json
+import forest_utils as utils
 
+def main():
+    parser = argparse.ArgumentParser(description='Misc utils')
+    parser.add_argument('--start', help='Start full service', action='store_true')
+    args = parser.parse_args()
+    if args.start:
+        print('starting full service')
+        asyncio.run(start(self=None))
 
-def start(self):
-    self.wallet_path.mkdir(parents=True, exists_ok=True)
+async def start(self):    
+    #self.wallet_path.mkdir(parents=True, exists_ok=True)
     cmd = ' '.join([
-        f'&& {constants.FULLSERVICE_DIR}/target/release/full-service',
+        f'&& {utils.get_secret("FULLSERVICE_DIR")}/target/release/full-service',
```

Comment:
this will allow things that would go in constants.py to be set locally as an envar or in a simple text file called `dev_secrets`